react-dataflow-dynamic/src/BrushableScatterplot.js

```diff
@@ -0,0 +1,49 @@
+import {useEffect, useRef, useState} from "react";
+
+function BrushableScatterplot({height, setSelection}) {
+  const ref = useRef();
+  const [module, setModule] = useState();
+
+  useEffect(() => {
+    async function load() {
+      // Dynamically load the Runtime and the latest published version of the
+      // notebook, https://observablehq.com/@d3/brushable-scatterplot.
+      const {Runtime, Inspector} = await import("https://cdn.jsdelivr.net/npm/@observablehq/runtime@4/dist/runtime.js");
+      const {default: notebook} = await import("https://api.observablehq.com/@d3/brushable-scatterplot.js?v=3");
```

Comment:
You’ll want to parallelize these using Promises.all:

```
const [{Runtime, Inspector}, {default: notebook}] = await Promise.all(
  import("https://cdn.jsdelivr.net/npm/@observablehq/runtime@4/dist/runtime.js"),
  import("https://api.observablehq.com/@d3/brushable-scatterplot.js?v=3")
);
```

Reply:
i changed to install the runtime statically at compile time and only import notebook at runtime… but i will remember this in the future!!